src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -28,51 +41,119 @@ public sealed partial class ProcessStartInfo
+        /// <format type="text/markdown"><![CDATA[
+        /// [!code-cpp[Process.Start_static#3](~/samples/snippets/cpp/VS_Snippets_CLR/Process.Start_static/CPP/processstartstatic3.cpp)]
+        /// [!code-csharp[Process.Start_static#3](~/samples/snippets/csharp/VS_Snippets_CLR/Process.Start_static/CS/processstartstatic3.cs)]
+        /// [!code-vb[Process.Start_static#3](~/samples/snippets/visualbasic/VS_Snippets_CLR/Process.Start_static/VB/processstartstatic3.vb)]
+        /// ]]></format>
+        /// <format type="text/markdown"><![CDATA[
+        /// [!code-cpp[Process.Start_static#2](~/samples/snippets/cpp/VS_Snippets_CLR/Process.Start_static/CPP/processstartstatic2.cpp)]
+        /// [!code-csharp[Process.Start_static#2](~/samples/snippets/csharp/VS_Snippets_CLR/Process.Start_static/CS/processstartstatic2.cs)]
+        /// [!code-vb[Process.Start_static#2](~/samples/snippets/visualbasic/VS_Snippets_CLR/Process.Start_static/VB/processstartstatic2.vb)]
+        /// ]]></format></example>
         public string Arguments
         {
             get => _arguments ?? string.Empty;
             set => _arguments = value;
         }
 
+        /// <summary>Gets a collection of command-line arguments to use when starting the application. Strings added to the list don't need to be previously escaped.</summary>
+        /// <value>A collection of command-line arguments.</value>
+        /// <remarks>The <see cref="System.Diagnostics.ProcessStartInfo.ArgumentList" /> and <see cref="System.Diagnostics.ProcessStartInfo.Arguments" /> properties are independent of one another and <b>only one of them can be used at the same time</b>. The main difference between both APIs is that <see cref="System.Diagnostics.ProcessStartInfo.ArgumentList" /> takes care of escaping the provided arguments and <b>internally</b> builds a single string that is passed to operating system when calling <see cref="System.Diagnostics.Process.Start(ProcessStartInfo)" />. So if you are not sure how to properly escape your arguments, you should choose <see cref="System.Diagnostics.ProcessStartInfo.ArgumentList" /> over <see cref="System.Diagnostics.ProcessStartInfo.Arguments" />.
+        /// </remarks>
+        /// <example>This example adds three arguments to the process start info.
+        /// <code class="lang-csharp">
+        /// var info = new System.Diagnostics.ProcessStartInfo("cmd.exe");
+        /// info.ArgumentList.Add("/c");
+        /// info.ArgumentList.Add("dir");
+        /// info.ArgumentList.Add(@"C:\Program Files\dotnet"); // there is no need to escape the space, the API takes care of it
+        /// // or if you prefer collection property initializer syntax:
+        /// var info = new System.Diagnostics.ProcessStartInfo("cmd.exe")
+        /// {
+        /// ArgumentList = {
+        /// "/c",
+        /// "dir",
+        /// @"C:\Program Files\dotnet"
+        /// }
```

Comment:
does code samples provided in the comments support indentation?
```suggestion
        /// ArgumentList = {
        ///     "/c",
        ///     "dir",
        ///     @"C:\Program Files\dotnet"
        /// }
```